src/block/icon-label/style.scss

```diff
@@ -1,8 +1,20 @@
 .stk-block-icon-label {
-	// Default width of the icon width.
-	.stk-block-icon {
-		flex: 0 0 64px;
+	// For old icon gap
```

Comment:
can you move this css to the bottom (isolating it from the rest of the "current" css code) and then label it as "deprecated"